.editorconfig

```
@@ -6,4 +6,4 @@ trim_trailing_whitespace = true
 end_of_line = lf
 indent_style = space
 insert_final_newline = true
-indent_size = 2
\ No newline at end of file
+indent_size = 4
```

Or do [what we do at ruff](https://github.com/astral-sh/ruff/blob/20217e9bbda5434288d96454e90cb0846c1f330f/.editorconfig#L11-L14)?

```suggestion
indent_size = 2

[*.{rs,py,pyi}]
indent_size = 4
```

---

Fair enough. PR updated to just be a copy of what's in Ruff.